src/_data/sidenav.yml

```diff
@@ -64,8 +64,6 @@
   permalink: /ui
   match-page-url-exactly: true
   children:
-    - title: Introduction to widgets
-      permalink: /ui/widgets-intro
```

Comment:
How about keep this for now, at least until I can finalize breaking up the UI category?

Perhaps with "Introduction" as the title?